dev/scripts/docs-to-google-sheets.js

```diff
@@ -0,0 +1,126 @@
+const path = require('path');
```

Comment:
We should use typescript always. We have already rewrite helper scripts from js to ts. Why do we write using javascript here?